.devops/deploy-pipelines.yml

```
@@ -130,19 +133,17 @@ stages:
               slotName: 'production'
             displayName: Deploy to io-p-fn-elt slot
 
-  # Check that the io-p-fn-elt instance is healthy
+  # Check that the staging instance is healthy
   - stage: Healthcheck
+    pool:
+      name: io-prod-linux
```

```suggestion
      name: $(AGENT_POOL)
```
this could be replaced with env var

---

The first time we run it's necessary to set it as string. Unfortunately, I wasn't able to make it work

---

You are right, but we can do it without merging the literal value:
https://pagopa.atlassian.net/wiki/spaces/IC/pages/84902884/Troubleshooting#L%E2%80%99healthcheck-non-funziona%2C-errore%3A-Pipeline-does-not-have-permissions-to-use-the-referenced-pool(s)-io-prod-linux.-For-authorization-details%2C-%E2%80%A6

---

I know, I wrote it :)

---

The problem is I wasn't able to run it successfully on this branch.

---

ok, let's do it with 2 PR then!